src/ledger/test/LedgerCloseMetaStreamTests.cpp

```diff
@@ -421,8 +421,8 @@ TEST_CASE_VERSIONS("meta stream contains reasonable meta", "[ledgerclosemeta]")
                             .bucketListWindowSamplePeriod = 1;
                     });
 
-                // Modify Soroban network config closes a ledger
-                ++targetSeq;
+                // Modify Soroban network config closes 2 ledgers
```

Comment:
`modifySorobanNetworkConfig` will close two ledgers because lcl + 1 is a merge ledger right? If this test was modified where targetSeq == 7 instead of 6 before `modifySorobanNetworkConfig` is called, then we'd need to revert the `+=2` change?

Reply:
Yeah, now `modifySorobanNetworkConfig` closes between 1 or 2 ledgers depending on the current lcl instead of just always closing one ledger.